bd/helloworld.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE hello_world (
```

Comment:
Rename this table to **dashboard** (Since we are creating a DashboardController in Laravel)